spec/helpers/commodities_helper_spec.rb

```diff
@@ -157,4 +157,21 @@
 
     it { is_expected.to eql 'commodity-ancestors__ancestor-23' }
   end
+
+  describe '#preferential_treatment_partial_for' do
+    subject { helper.preferential_treatment_partial_for(geographical_area_id) }
+
+    context 'when the country has preferential treatment in locals' do
+      let(:geographical_area_id) { 'UA' }
+
+      it { is_expected.to be_html_safe }
+      it { is_expected.to be_present }
+    end
+
+    context 'when the country does NOT have preferential treatment in locals' do
+      let(:geographical_area_id) { 'IT' }
+
+      it { is_expected.to be_empty }
+    end
+  end
```

Comment:
Now that we are using locales, test tests are less valuable.
I am thinking to remove them.

Reply:
I'm not actually sure a helper method makes sense for this anymore